src/Models/LoginSession.php

```diff
@@ -96,6 +97,14 @@ class LoginSession extends DataObject
      */
     private static $default_session_lifetime = 3600;
 
+    /**
+     * The length of time between two updates to the LastAccessed field
+     *
+     * @config
```

Comment:
```suggestion
```
The `@config` annotation is effectively meaningless - all `private static` properties are config unless they have the `@internal` annotation.